package.json

```diff
@@ -60,7 +60,8 @@
     "react-native-svg": "15.2.0",
     "react-native-svg-charts": "^5.4.0",
     "react-native-web": "~0.19.6",
-    "typescript": "~5.3.3"
+    "typescript": "~5.3.3",
+    "underscore": "^1.13.6"
```

Comment:
Are we trying to add in typescript into the codebase? additionally, is underscore a new library we are also introducing? I see its used for the Debouncing functionallity but could its functionality be useful elsewhere? do you know?

Reply:
since we squash merge the entirety of layout into main before, I can't really traceback to why we added typescript. As far as I know, it's just a dependency of a lot of libraries, but not necessary to be in package.json. As for underscore, there's a lot of utilities in it. As far as the debouncing PRs, I'm using `debounce` and `once` from it. Maybe we can use more. Really good questions tho!

Reply:
typescript is a dependency for prettier